### DISCUSSION TITLE
💬 请问下载多个BV号视频该如何操作呢？

### DISCUSSION THREAD

**Discussion by WellFolg** [yutto-dev/yutto#6] [Q&A] [answered]
例如下载不同UP的投稿视频，另外可否自定义下载的文件名(不包括后缀)？

**Comment by SigureMo** [comment]
> 另外可否自定义下载的文件名(不包括后缀)？

可以的，这一点可以通过使用“子路径模板”功能来进行自定义，下载单个投稿视频时默认提供的模版是 "{title}"，因此会自动使用标题作为视频名称。花括号中的是模板变量，如果你不需要模板的话，直接使用字符串也是可以的，比如像下面这样：

```bash
yutto BV1vZ4y1M7mQ -tp "video"
```

这样就会自动生成 video.mp4 了（如果添加了 --audio-only 的话后缀自然会是 .aac）

**Comment by SigureMo** [comment, marked as answer]
> 下载不同UP的投稿视频

如果想要下载多个 BV 号视频当前只能使用文件列表的方式

比如创建一个 `videos.txt`，内容如下：

```
av1
av2
av3
```

那么只需要 `yutto videos.txt` 即可下载它们。

当然如果你熟悉 shell 的话，写 shell 脚本循环调用 yutto 也是可以的。

```bash
for avid in av1 av2 av3
do
    yutto $avid
done
```
